homework_create_more_models/README.md

```diff
@@ -0,0 +1,269 @@
+    Migrations for 'blog':
+      0002_comment.py:
+        - Create model Comment
+
+You can see that this command created for us another migration file in `blog/migrations/` directory. Now we need to apply those changes with `python manage.py migrate blog`. It should look like this:
+
+    (myvenv) ~/djangogirls$ python manage.py migrate blog
+    Operations to perform:
+      Apply all migrations: blog
+    Running migrations:
+      Rendering model states... DONE
+      Applying blog.0002_comment... OK
+
+Our Comment model exists in database now. It would be nice if we had access to it in our admin panel.
+
+## Register comment model in admin panel
+
+To register model in admin panel, go to `blog/admin.py` and add line:
+
+    admin.site.register(Comment)
+
+Don't forget to import Comment model, file should look like this:
+
+    from django.contrib import admin
+    from .models import Post, Comment
+
+    admin.site.register(Post)
+    admin.site.register(Comment)
+
+If you type `python manage.py runserver` in command prompt and go to [http://127.0.0.1:8000/admin/](http://127.0.0.1:8000/admin/) in your browser, you should have access to list, add and remove comments. Don't hesitate to play with it!
+
+## Make our comments visible
+
+Go to `blog/templated/blog/post_detail.html` file and add those lines before `{% endblock %}` tag:
+
+	<hr>
+	{% for comment in post.comments.all %}
+		<div class="comment">
+			<div class="date">{{ comment.created_date }}</div>
+			<strong>{{ comment.author }}</strong>
+			<p>{{ comment.text|linebreaks }}</p>
+		</div>
+	{% empty %}
+		<p>No comments here yet :(</p>
+	{% endfor %}
+
+Now we can see the comments section on pages with post details.
+
+But it can look a little bit better, add those some css to `static/css/blog.css`:
+
+    .comment {
+        margin: 20px 0px 20px 20px;
+    }
+
+We can also let know about comments on post list page, go to `blog/templates/blog/post_list.html` file and add line:
+
+    <a href="{% url 'blog.views.post_detail' pk=post.pk %}">Comments: {{ post.comments.count }}</a>
+
+After that our template should look like this:
+
+    {% extends 'blog/base.html' %}
+
+    {% block content %}
+        {% for post in posts %}
+            <div class="post">
+                <div class="date">
+                    {{ post.published_date }}
+                </div>
+                <h1><a href="{% url 'blog.views.post_detail' pk=post.pk %}">{{ post.title }}</a></h1>
+                <p>{{ post.text|linebreaks }}</p>
+                <a href="{% url 'blog.views.post_detail' pk=post.pk %}">Comments: {{ post.comments.count }}</a>
+            </div>
+        {% endfor %}
+    {% endblock content %}
+
+## Let your readers write comments
+
+Right now we can see comments on our blog, but we cannot add them, let's change that!
+
+Go to `blog/forms.py` and add those lines to the end of the file:
+
+    class CommentForm(forms.ModelForm):
+
+    	class Meta:
+    		model = Comment
```

Comment:
Indentation again, just check all the code blocks, I won't repeat the comment about indentation any more, you know what's it all about :) :girl: :sheep: :steam_locomotive: